modules/ROOT/pages/index.adoc

```diff
@@ -1,12 +1,17 @@
 = What's New
 
 == Highlights
+Two Java build options are available at this release:
 
-=== Custom Conflict Resolution
+*  Java Android - enabling development and deployment of Java apps in an android environment (Read more: xref:java-android.adoc[Java Android])
+* Java Platform
+ - enabling development and deployment of Java apps in any environment that supports the JVM model.
```

Comment:
Not sure if this list item is intentionally different than the one the precedes it, but it renders as a sub-bullet and doesn't look like it's formatted correctly.